docs/pi_setup.md

```diff
@@ -40,74 +65,95 @@ rsn_pairwise=CCMP
 
-1. Run dora, You can see dora's options with `dora --help`, you likely need to edit the config file. `dora`'s config is in a format that's easy to generate programmatically, not with manual editing as the first priority. zzRemember to specify an `interfaces` section in the config.yaml so hostapd and dora don't use the same interface.
+1. Run dora, You can see dora's options with `dora --help`, you likely need to edit the config file. `dora`'s config is in a format that's easy to generate programmatically, not with manual editing as the first priority. Remember to specify an `interfaces` section in the config.yaml so hostapd and dora don't use the same interface.
 
-A very simple config that matches how this guide has configured hostapd might look like (see [example.yaml](../example.yaml) for the full set of options):
+A very simple config (IPv4 only) that matches how this guide has configured hostapd might look like:
 
 ```yaml
-interfaces:
-    - wlan0
+interfaces: 
+  - wlan0
 networks:
     192.168.5.0/24:
-        authoritative: true
         probation_period: 86400
+        server_id: 192.168.5.1
         ranges:
             - start: 192.168.5.2
               end: 192.168.5.250
               config:
                   lease_time:
                       default: 3600
+                      min: 1200
+                      max: 4800
               options:
                   values:
-                      3: # router
+                      1: # subnet mask (if not specified, comes from `interfaces`)
+                          type: ip
+                          value: 255.255.255.0
+                      3: # router (if not specified, will come from `interfaces`)
                           type: ip_list
                           value:
                               - 192.168.5.1
-                      6: # domain name (if running a DNS server like dnsmasq also, use it's IP)
+                      6: # domain name (if running a DNS server like dnsmasq also, use its IP)
                           type: ip_list
                           value:
                               - 8.8.8.8
+                      28: # broadcast addr (if not specified, comes from `interfaces`)
+                         type: ip
+                         value: 192.168.5.255
 ```
 
-(use --help to see `dora` arguments)
+You may wish to save this minimal config to `pi.yaml` to try it out, or see [example.yaml](../example.yaml) for the full set of options. You can also use `dora --help` to see arguments.
 
 Run dora:
 
+After you have saved the above minimal config to `pi.yaml`, you should be able to run the following (assuming you compiled a release binary with `armv7-unknown-linux-gnueabihf` as a target earlier, if not please alter the path to your `dora` bin appropriately):
+
 ```
-sudo DORA_LOG="debug" ./dora -c example.yaml -d em.db
+sudo DORA_LOG="debug" target/armv7-unknown-linux-gnueabihf/release/dora -c pi.yaml -d em.db
```

Comment:
If you followed this guide to the letter, you'd have cross-compiled a binary and copied to the pi, so this path won't exist.

Reply:
I do agree with your point here. This info has been moved to a more relevant section of the README and a note was added about ensuring that `./dora` refers to your binary path.